algo/src/test/java/com/hartwig/serve/sources/ckb/ActionableTrialFactoryTest.java

```diff
@@ -11,37 +11,48 @@
 import com.hartwig.serve.ckb.datamodel.clinicaltrial.ClinicalTrial;
 import com.hartwig.serve.ckb.datamodel.clinicaltrial.Location;
 import com.hartwig.serve.ckb.datamodel.clinicaltrial.VariantRequirementDetail;
+import com.hartwig.serve.datamodel.DatamodelTestFactory;
 import com.hartwig.serve.datamodel.EvidenceDirection;
 import com.hartwig.serve.datamodel.EvidenceLevel;
 import com.hartwig.serve.datamodel.Knowledgebase;
+import com.hartwig.serve.sources.ckb.blacklist.CkbBlacklistStudyTest;
+import com.hartwig.serve.sources.ckb.blacklist.CkbStudyBlacklistModel;
 
 import org.apache.commons.compress.utils.Sets;
+import org.apache.logging.log4j.util.Strings;
 import org.jetbrains.annotations.NotNull;
 import org.junit.Test;
 
 public class ActionableTrialFactoryTest {
 
+    private static final CkbStudyBlacklistModel blacklistStudy = CkbBlacklistStudyTest.createCkbBlacklistStudyEmpty();
```

Comment:
An empty model should be created from a helper test factory rather than another unit test

Also, private static final variables should be capitalised (`BLACKLIST_MODEL`)

Reply:
done